[2022-12-22T11:13:54.616+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: co2balance.RunAPIService manual__2022-12-22T11:13:37.008743+00:00 [queued]>
[2022-12-22T11:13:54.638+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: co2balance.RunAPIService manual__2022-12-22T11:13:37.008743+00:00 [queued]>
[2022-12-22T11:13:54.638+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2022-12-22T11:13:54.638+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 1
[2022-12-22T11:13:54.639+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2022-12-22T11:13:54.673+0000] {taskinstance.py:1304} INFO - Executing <Task(PythonOperator): RunAPIService> on 2022-12-22 11:13:37.008743+00:00
[2022-12-22T11:13:54.683+0000] {standard_task_runner.py:55} INFO - Started process 4483 to run task
[2022-12-22T11:13:54.687+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'co2balance', 'RunAPIService', 'manual__2022-12-22T11:13:37.008743+00:00', '--job-id', '103', '--raw', '--subdir', 'DAGS_FOLDER/co2balance.py', '--cfg-path', '/tmp/tmprmykc2kl']
[2022-12-22T11:13:54.690+0000] {standard_task_runner.py:83} INFO - Job 103: Subtask RunAPIService
[2022-12-22T11:13:54.816+0000] {task_command.py:389} INFO - Running <TaskInstance: co2balance.RunAPIService manual__2022-12-22T11:13:37.008743+00:00 [running]> on host 1a27c7ed0719
[2022-12-22T11:13:54.934+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=CO2balance
AIRFLOW_CTX_DAG_ID=co2balance
AIRFLOW_CTX_TASK_ID=RunAPIService
AIRFLOW_CTX_EXECUTION_DATE=2022-12-22T11:13:37.008743+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-12-22T11:13:37.008743+00:00
[2022-12-22T11:13:54.938+0000] {logging_mixin.py:137} WARNING - INFO:     Will watch for changes in these directories: ['/opt/***']
[2022-12-22T11:13:54.939+0000] {logging_mixin.py:137} WARNING - INFO:     Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)
[2022-12-22T11:13:54.939+0000] {logging_mixin.py:137} WARNING - INFO:     Started reloader process [4483] using StatReload
[2022-12-22T11:13:54.939+0000] {taskinstance.py:1772} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 192, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/app/api_service/main.py", line 47, in run
    uvicorn.run("co2balance_api_service:app", port=8000, reload=True)
  File "/home/airflow/.local/lib/python3.7/site-packages/uvicorn/main.py", line 564, in run
    ChangeReload(config, target=server.run, sockets=[sock]).run()
  File "/home/airflow/.local/lib/python3.7/site-packages/uvicorn/supervisors/basereload.py", line 44, in run
    self.startup()
  File "/home/airflow/.local/lib/python3.7/site-packages/uvicorn/supervisors/basereload.py", line 80, in startup
    self.process.start()
  File "/usr/local/lib/python3.7/multiprocessing/process.py", line 110, in start
    'daemonic processes are not allowed to have children'
AssertionError: daemonic processes are not allowed to have children
[2022-12-22T11:13:54.956+0000] {taskinstance.py:1327} INFO - Marking task as FAILED. dag_id=co2balance, task_id=RunAPIService, execution_date=20221222T111337, start_date=20221222T111354, end_date=20221222T111354
[2022-12-22T11:13:54.976+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 103 for task RunAPIService (daemonic processes are not allowed to have children; 4483)
[2022-12-22T11:13:55.019+0000] {local_task_job.py:159} INFO - Task exited with return code 1
[2022-12-22T11:13:55.109+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
